blocks/api/post.pegjs

```diff
@@ -1,8 +1,14 @@
 {
 
+function untransformValue( value ) {
+	return 'string' === typeof value
+		? value.replace( /\\-/g, '-' )
```

Comment:
Also, see #1088 for background on the issues with a double-hyphen in an HTML comment